client/profile/user-profile-overlay.tsx

```diff
@@ -236,9 +237,9 @@ function OverlayContents({ userId, onDismiss }: { userId: SbUserId; onDismiss: (
         <>
           <div>
             <SectionHeader>Info</SectionHeader>
-            <Body1 title={longTimestamp.format(profile.created)}>
-              Joined {joinDateFormat.format(profile.created)}
-            </Body1>
+            <Tooltip text={longTimestamp.format(profile.created)}>
```

Comment:
imo the aria-label here should be something more like `Joined ${longFormattedDate}`, I believe once you add that attribute it's going to ignore the children of this element so it should reproduce the full thing if possible

Reply:
Can you clarify what you mean by "once you add that attribute it's going to ignore the children of this element"?

Reply:
As in, the things using the ARIA attributes (e.g. screenreaders) will read out the `aria-label` rather than the text content of the children

Reply:
Probably I guess we should be marking the Portal content as `aria-hidden` as well, since it doesn't follow the normal document hierarchy

Reply:
http://whatsock.com/training/matrices/#:~:text=3.1.11%3A%20Tooltip%20Role maybe this is a better solution tbh, I may just try to implement this so we can get this checked in without further back-and-forth